lib/cc/analyzer/issue_relative_path_validation.rb

```diff
@@ -11,6 +11,13 @@ def message
 
       private
 
+      def relative_to?(directory)
```

Comment:
Updated the expand logic to use `Pathname#realpath` which has the same cleaning behavior as `cleanpath` but also follows symlinks.